src/os/code_3d330_len_990.c

```diff
@@ -1,13 +1,44 @@
 #include "common.h"
 
-void INCLUDE_ASM(os/code_3d330_len_990, guFrustumF, float mf[4][4], float l, float r, float b, float t, float n,
-                 float f, float scale);
+void INCLUDE_ASM(
```

Comment:
These are yucky

Reply:
Personally I think they're.....fine. We could change the function param breaking style to something more compact if you'd prefer? Or selectively disable clang-format for this line.